subprojects/docs/src/docs/release/notes.md

```diff
@@ -243,13 +239,11 @@ For more information, see [Exporting keys](userguide/dependency_verification.htm
 
 #### Consistent task execution for configuration cache hit and configuration cache miss builds
```

Comment:
@adammurdoch I modified your this section. Can you make sure it still makes sense? Feel free to change as needed